.coveragerc

```diff
@@ -1,5 +1,9 @@
 [report]
-omit = */samples/*
+omit =
+    */samples/*
+    # Don't report coverage over platform-specific modules.
+    oauth2client/contrib/_fcntl_opener.py
```

Comment:
Yes? Do you want to? As soon as multistore file has 100% coverage, I'm swapping out all of this for fasteners.